src/style.css

```diff
@@ -0,0 +1,109 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 900;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin: 20px auto 0;
+}
+
+.search-form--big {
+  height: 70px;
+}
+
+.search-form--small {
+  height: 42px;
+}
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-form__label:after {
+  content: "";
+  position: absolute;
+  display: block;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.search-form__label--big:after {
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search-form__label--small:after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search-form__input {
+  box-sizing: border-box;
```

Comment:
let's set `box-sizing` to the `body`

Reply:
If I set the 'box-sizing: border box' for 'body', I don't pass the tests

Reply:
try to set with `*` selector
`* {
  box-sizing: border-box
}`